src/CommonDBRelation.php

```diff
@@ -1702,8 +1706,8 @@ public static function processMassiveActionsForOneItemtype(
                             && ($item_1->getType() == $item_2->getType())
                         ) {
                             $ORWHERE = [
-                                static::$items_id_1 = $item_2->getID(),
-                                static::$items_id_2 = $item_2->getID(),
+                                static::$items_id_1 => $item_2->getID(),
+                                static::$items_id_2 => $item_2->getID(),
```

Comment:
This one was very bad, values were assigned into the static variables instead of being set as array key/values pairs.

Reply:
Please open a decicated PR for this change.